source/Generic/SplashScreen/SplashScreen.cs

```diff
@@ -331,48 +331,59 @@ private void SplashWindowClosed(object sender, EventArgs e)
+            const string microVideoName = "VideoMicrotrailer.mp4";
 
-            var splashVideo = string.Format(videoPathTemplate, "games", game.Id.ToString());
+            var baseVideoPathTemplate = Path.Combine(PlayniteApi.Paths.ConfigurationPath, "ExtraMetadata", "{0}", "{1}");
+            
+            var baseSplashVideo = string.Format(baseVideoPathTemplate, "games", game.Id.ToString());
+            var splashVideo = Path.Combine(baseSplashVideo, videoIntroName);
             if (FileSystem.FileExists(splashVideo))
             {
                 logger.Info(string.Format("Specific game video found in {0}", splashVideo));
-                return splashVideo;
+                return LogAcquiredVideo("Game-specific", splashVideo);
             }
 
-            if (game.Source != null)
+            var isDesktop = PlayniteApi.ApplicationInfo.Mode is ApplicationMode.Desktop;
+            if (ShouldUseMicroTrailer(isDesktop))
             {
-                splashVideo = string.Format(videoPathTemplate, "sources", game.Source.Id.ToString());
+                splashVideo = Path.Combine(baseSplashVideo, microVideoName);
                 if (FileSystem.FileExists(splashVideo))
                 {
-                    logger.Info(string.Format("Source video found in {0}", splashVideo));
-                    return splashVideo;
+                    return LogAcquiredVideo("Micro trailer", splashVideo);
                 }
             }
 
-            if (game.Platforms.HasItems())
+            var videoPathTemplate = Path.Combine(baseVideoPathTemplate, videoIntroName);
+
+            if (game.Source != null)
             {
-                splashVideo = string.Format(videoPathTemplate, "platforms", game.Platforms[0].Id.ToString());
+                splashVideo = string.Format(videoPathTemplate, "sources", game.Source.Id.ToString());
                 if (FileSystem.FileExists(splashVideo))
                 {
-                    logger.Info(string.Format("Platform video found in {0}", splashVideo));
-                    return splashVideo;
+                    return LogAcquiredVideo($"Source '{game.Source.Name}'", splashVideo);
                 }
             }
 
-            if (PlayniteApi.ApplicationInfo.Mode == ApplicationMode.Desktop)
-            {
-                splashVideo = string.Format(videoPathTemplate, "playnite", "Desktop");
-            }
-            else
+            var platform = game.Platforms?.FirstOrDefault();
+            if (platform != null)
             {
-                splashVideo = string.Format(videoPathTemplate, "playnite", "Fullscreen");
+                splashVideo = string.Format(videoPathTemplate, "platforms", platform.Id.ToString());
+                if (FileSystem.FileExists(splashVideo))
+                {
+                    return LogAcquiredVideo($"Platform '{platform.Name}'", splashVideo);
+                }
             }
 
+            var mode = isDesktop
```

Comment:
I think it's fine to keep it in a single line as it's not long enough to require multiple lines

Reply:
Updated.